packages/slice-machine/lib/env/manifest.ts

```diff
@@ -49,6 +54,10 @@ export function extractRepo(parsedRepo: ParseResult): string | undefined {
   }
 }
 
+function validateFramework(framework?: string): boolean {
```

Comment:
It should probably part of the frameworkCodec or at least called from it so you can reuse it then in the Manifest Codec.